src/metabase/driver.clj

```diff
@@ -412,7 +412,10 @@
     :advanced-math-expressions
 
     ;; Does the driver support percentile calculations (including median)
-    :percentile-aggregations})
+    :percentile-aggregations
+
+    ;; Does this driver support pivot table pushdown? (executing a pivot table in a driver-specific way)
+    :pivot-table-pushdown})
```

Comment:
this is a placeholder for now that we can use later

Reply:
A better name would be some variation on `OLAP` as we'll eventually want to expose this beyond just pivot tables.